controllers/sriovibnetwork_controller.go

```diff
@@ -129,6 +129,13 @@ func (r *SriovIBNetworkReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			return reconcile.Result{}, err
 		}
 	}
+
+	netAttDef.Spec.Config, err = formatJSON(netAttDef.Spec.Config)
```

Comment:
nit: could you move this to after L119  so its in the same place where netAttDef is created.

Reply:
done. Thanks!